src/altinn-app-frontend/src/features/form/layout/update/updateFormLayoutSagas.ts

```diff
@@ -529,19 +533,6 @@ export function* watchInitialCalculatePageOrderAndMoveToNextPageSaga(): SagaIter
   }
 }
 
-export function* watchUpdateCurrentViewSaga(): SagaIterator {
```

Comment:
Husker denne ble lagt til for at backend-valideringer ikke skulle kjøre på utdatert skjemadata. Det var noe greier med at hvis viewet ble oppdatert før ulagrede data var lagret, så ville ikke valideringsmeldingen fra backend dukke opp ettesom den valideringen ikke hadde fått kjørt enda. Vet ikke hvordan dette funker nå, men kan være verdt å sjekke ut.
https://github.com/Altinn/app-frontend-react/pull/187

Reply:
Oooh! Spenstig - den delen av historien fikk ikke jeg med meg. Interessant! Kan forstå at det var vanskelig å lage en cypress-test for å verifisere at feilen var fikset den gang - men det hadde vært gunstig nå. De unit-testene som lå inne testet jo egentlig at _oppførselen til funksjonen var slik som den var implementert_, og det testet ikke at _oppførselen til app-frontend nå var som forventet_.

Jeg prøvde å beholde oppførselen (ved å legge inn `waitFor`) slik at man fortsatt ikke navigerer til neste side før data er lagret, men dette bør testes skikkelig tenker jeg. Jeg prøver å lage en relevant cypress-test for dette.